pkg/pods/pod.go

```diff
@@ -593,6 +596,34 @@ func (pod *Pod) Launchables(manifest Manifest) ([]launch.Launchable, error) {
 	return launchables, nil
 }
 
+func (pod *Pod) SetFinishExec(finishExec []string) {
+	var runAs string
+
+	if manifest, err := pod.CurrentManifest(); err == nil {
```

Comment:
what does it mean to have an `err` here? does that mean the pod is being deployed for the first time? seems weird that the user would be different on a first time deploy than a second time deploy.